ws-server/src/server/module.rs

```diff
@@ -47,7 +47,15 @@ impl RpcModule {
 			Box::new(move |id, params, tx, _| {
 				match callback(params) {
 					Ok(res) => send_response(id, tx, res),
-					Err(InvalidParams) => send_error(id, tx, JsonRpcErrorCode::InvalidParams.into()),
+					// TODO: this looks wonky...
+					Err(CallError::InvalidParams(InvalidParams)) => {
```

Comment:
I think we can remove this `InvalidParams` type and just use to `InvalidParams` variant,

perhaps link to https://github.com/paritytech/jsonrpsee/issues/299

Reply:
thus, if CallError would implement a trait or something `JsonRpcErrorCode` then you could just to `err.to_error_code()` or something similar